tests/test_generator.py

```diff
@@ -51,37 +54,39 @@ def testCcdVisit(self):
             "CcdVisit": {
                 "columns": {"ccdVisitId": columns.VisitIdGenerator(),
                             "filterName": columns.FilterGenerator(filters="ugr"),
-                            "ra,decl": columns.RaDecGenerator(chunker)
-                            }
+                            "ra,decl": columns.RaDecGenerator()
+                            },
+                "density": UniformSpatialModel(50),
             },
         }
         chunk_id = 5000
         seed = 1
         edge_width = 0.017 # degrees
-        generator = DataGenerator(generator_spec)
-        chunk_table = generator.make_chunk(chunk_id, num_rows=50, seed=seed,
-                                           edge_width=edge_width, edge_only=False)
+        generator = DataGenerator(generator_spec, chunker, seed=seed)
+        chunk_area = chunker.getChunkBounds(chunk_id).getArea() / RAD_PER_DEG**2
+        chunk_table = generator.make_chunk(chunk_id, edge_width=edge_width, edge_only=False)
         self.assertIn("CcdVisit", chunk_table.keys())
-        self.assertEqual(len(chunk_table["CcdVisit"]), 50)
+        self.assertAlmostEqual(len(chunk_table["CcdVisit"]), int(50 * chunk_area), delta=1)
```

Comment:
I'm confused as to why CcdVisit has anything to do with object density.

Reply:
Right, this is now removed since visits are precomputed.